### PR TITLE
Initial Commit & File Creation

### PR DESCRIPTION
**Pull request by Rachella**
Initial set-up of files. Created README, LICENSE, scripts, html and css folders and files.

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+section, h1 {
```
Comment (h8rry) [inline css/custom.css]: Indentations are off

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+bower_components
```
Comment (h8rry) [inline .gitignore]: The best practice is to leave a line break at the end of each file.

**Comment by h8rry** [comment]
Note: you can check your file changes here https://github.com/WDI-HK-8/UrbanFictionary/pull/1/files

File: index.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+          <h3> Question 1 </h3>
+            <div class="pane">
+              <h4>Just Ducky</h4>
+              <ul>
+                <li>Choice A</li>
+                <li>Choice B</li>
+                <li>Choice C</li>
+              </ul>
+            </div>
+          <h3> Question 2 </h3>
```
Comment (h8rry) [inline index.html]: Further reference: We should generate / append questions dynamically, instead of `hard coding` them

File: index.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+<!-- Game Page  -->
+    <div class="container" id="page">
+      <div class="row">
+  <!-- Player Details     -->
+        <div class="col-xs-3">
+
+        </div>
+  <!-- Main Game       -->
+        <div class="col-xs-6 accordion" id="Q1">
```
Comment (h8rry) [inline index.html]: `id` and `class` namings should to lowercase

File: js/application.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (h8rry) [inline js/application.js]: Indentations please :)

**Comment by h8rry** [comment]
The rest looks fine. Please slack me when you've fixed the stuff :)

File: .gitignore (hunk 2 above)
Comment (Rachella) [inline .gitignore]: Thanks, have added .. I think.

File: js/application.js (hunk 5 above)
Comment (Rachella) [inline js/application.js]: Oh I thought there was no indentation for first line - okay, indented. Thanks!

Comment (h8rry) [inline js/application.js]: I meant that please fix the indentations for the rest of the file. The
first line shouldn't have indentations

Comment (Rachella) [inline js/application.js]: removed indentation from first line. Indented second line. No indent for closing tag.

**Comment by h8rry** [comment]
Looks good!

**Pull request merged**